source/retryable-writes/tests/README.md

```diff
@@ -106,17 +106,197 @@ Drivers should test that transactions IDs are always included in commands for su
 
 The following tests ensure that retryable writes work properly with replica sets and sharded clusters.
 
-1. Test that retryable writes raise an exception when using the MMAPv1 storage engine. For this test, execute a write
-   operation, such as `insertOne`, which should generate an exception. Assert that the error message is the replacement
-   error message:
+- Test that retryable writes raise an exception when using the MMAPv1 storage engine. For this test, execute a write
+  operation, such as `insertOne`, which should generate an exception. Assert that the error message is the replacement
+  error message:
 
-   ```
-   This MongoDB deployment does not support retryable writes. Please add
-   retryWrites=false to your connection string.
-   ```
+  ```
+  This MongoDB deployment does not support retryable writes. Please add
+  retryWrites=false to your connection string.
+  ```
 
-   and the error code is 20.
+  and the error code is 20.
 
-   [!NOTE]
-   storage engine in use MAY skip this test for sharded clusters, since `mongos` does not report this information in its
-   `serverStatus` response.
+  > [!NOTE]
+  > Drivers that rely on `serverStatus` to determine the storage engine in use MAY skip this test for sharded clusters,
+  > since `mongos` does not report this information in its `serverStatus` response.
+
+- Test that drivers properly retry after encountering PoolClearedErrors. This test MUST be implemented by any driver
+  that implements the CMAP specification.
+
+  This test requires MongoDB 4.3.4+ for both the `errorLabels` and `blockConnection` fail point options.
+
+  - Create a client with maxPoolSize=1 and retryWrites=true. If testing against a sharded deployment, be sure to connect
```

Comment:
These steps were also numeric in the original content. I think it's worth preserving those, as some drivers may refer back to numbered steps from comments within their prose tests.

Reply:
I had been fighting with the formatter, fixed now.